### ISSUE TITLE
deploy fails on Python 3

### ISSUE THREAD

**Issue by davidism** [lektor/lektor#424]
I switched [pallets/website](https://github.com/pallets/website) to build with Python 3.6 on Travis. The build fails with `TypeError: can't concat str to bytes`. This should fix the specific error, but there may be other occurrences of the issue.

```diff
- target.append(username.encode('utf-8') + '@')
+ target.append(username.encode('utf-8') + b'@')
```

```pytb
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.6.2/bin/lektor", line 11, in <module>
    sys.exit(main())
  File "/home/travis/virtualenv/python3.6.2/lib/python3.6/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/home/travis/virtualenv/python3.6.2/lib/python3.6/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/home/travis/virtualenv/python3.6.2/lib/python3.6/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/travis/virtualenv/python3.6.2/lib/python3.6/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/travis/virtualenv/python3.6.2/lib/python3.6/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/home/travis/virtualenv/python3.6.2/lib/python3.6/site-packages/click/decorators.py", line 64, in new_func
    return ctx.invoke(f, obj, *args[1:], **kwargs)
  File "/home/travis/virtualenv/python3.6.2/lib/python3.6/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/home/travis/virtualenv/python3.6.2/lib/python3.6/site-packages/lektor/cli.py", line 301, in deploy_cmd
    for line in event_iter:
  File "/home/travis/virtualenv/python3.6.2/lib/python3.6/site-packages/lektor/publisher.py", line 231, in publish
    client = self.get_command(target_url, tempdir, credentials)
  File "/home/travis/virtualenv/python3.6.2/lib/python3.6/site-packages/lektor/publisher.py", line 220, in get_command
    target.append(username.encode('utf-8') + '@')
TypeError: can't concat str to bytes
```

**Comment by nixjdm** [comment]
It looks like this error should already be fixed in master (by getting rid of the encoding), though it isn't in the latest release of 3.0.1 See in the commit that fixed it: https://github.com/lektor/lektor/commit/0768df17a22e199cf04532c44ac26f42e8723a4e

Hopefully we'll get this released soon. I can ping you when it is.

**Comment by dwt** [comment]
Please ping this bug report when released - I'm waiting for this too.

**Comment by nixjdm** [comment]
@davidism @dwt Sorry for the inconvenience and wait. This is now in the newely released Lektor 3.1
